### DISCUSSION TITLE
How to enable the horizontal line that tracks the scroll down percentage

### DISCUSSION THREAD

**Discussion by yyang768osu** [alshedivat/al-folio#557] [Q&A] [answered]
As in this site: https://andreea7b.github.io/, if you scroll down the page, the horizontal line just below the header will keep track of the scroll down percentage.

How do we enable it? 

Thank you!

**Comment by cmoralesmx** [comment]
Hi, I've seen the feature you mention in a number of featured sites, I am currently working on adding that functionality to the template but it will take a while as I am currently prioritizing other areas.
Thus, my motivation for replying to your post is to check if there is enough interest for this feature from more users of the template.
If that is the case then, I will increase the priority of this feature

**Comment by cmoralesmx** [comment, marked as answer]
Hi again, this feature is now part of the template
